eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/ReplicatedCassandra.scala

```diff
@@ -334,7 +334,11 @@ private[journal] object ReplicatedCassandra {
                                       for {
                                         segmentNrs <- from
                                           .toSegmentNr(segmentSize)
-                                          .to[F] { to.toSegmentNr(segmentSize) }
+                                          .to[F] {
+                                            // TODO MR why we don't use `to.next[Option].getOrElse(SegmentNr.max)` as in `def purge`?
```

Comment:
here `to` is `SeqNr` not `SegmentNr`.

The deletion should happened to exactly `to` rather next sequence number after `to`

Reply:
the full code is `to.toSegmentNr(segmentSize)`

my question is related to how `purge` works - it overshoots by one segment